docs/design/features/globalization-hybrid-mode.md

```diff
@@ -463,6 +471,11 @@ Affected public APIs:
 - String.IndexOf
 - String.LastIndexOf
 
+Methods that calculate `matchLength` throw PlatformNotSupported exception:
+[CompareInfo.IndexOf](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.compareinfo.indexof?view=system-globalization-compareinfo-indexof(system-readonlyspan((system-char))-system-readonlyspan((system-char))-system-globalization-compareoptions-system-int32@))
+
+[CompareInfo.LastIndexOf](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.compareinfo.lastindexof?view=system-globalization-compareinfo-lastindexof(system-readonlyspan((system-char))-system-readonlyspan((system-char))-system-globalization-compareoptions-system-int32@))
+
```

Comment:
Same for Apple, let's add info about `PlatformNotSupported` exception for `IsPrefix/IsSuffix ` also.